pkg/github/models/code.go

```diff
@@ -16,7 +16,7 @@ type Code struct {
     object_type *string
     // The object_url property
     object_url *string
-    // The property property
+    // The property
```

Comment:
I'm happy to drop this change... _generally_ duplicated words are errors, and often the proper fix is additional punctuation.